README.md

```diff
@@ -0,0 +1,13 @@
+# zlib [![CI][CI badge]][CI page] [![Hackage][Hackage badge]][Hackage page]
```

Comment:
Given that CIs come and go, and `README` is frozen forever on Hackage, I wonder whether we want to add CI badges. Moreover, this particular link will rot away once we change the name of GHA workflow (e. g., switching from `haskell-ci`-generated to `haskell/actions/setup`-based).

Reply:
The name is extracted from:

```
name: Haskell-CI
```
in the file. I don't understand what would prevent us from keeping it, if it's just something that we can set?

Reply:
Potentially we can, but this is one more thing to track and keep in mind. At some point we might end up with several GHA workflows. Shall we list all of them?.. What happens if the repo is moved under another organization?

Reply:
Another point is that it is weird when a badge in `README` for a fixed version on Hackage points to the last `HEAD` build. Failing `HEAD` build does not have any implications on the released version, but can cause such wrong impression.

This is not a big deal, if you do not feel convinced I'm not gonna press.

Reply:
I've given some thought to it, and considering that the README targets GitHub users first and foremost, and that Hackage builds the package itself, we don't really have to worry about this badge. But thanks for raising this point. :)